docs/en-US/component/text.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Text
+lang: en-US
+---
+
+# Text
+
+Used for text.
+
+## Basic
+
+:::demo Use the `type` attribute to define Text's type.
+
+text/basic
+
+:::
+
+## Sizes
+
+:::demo Use attribute `size` to set additional sizes with `large`, `default` or `small`.
+
+text/sizes
+
+:::
+
+## Ellipsis
+
+:::demo Pass the `truncated` prop to render an ellipsis when the text exceeds the width of the viewport or max-width set.
+
+text/truncated
+
+:::
+
+## Override
+
+:::demo Use attribute `tag` to override element
+
+text/override
+
+:::
+
+## Mixed
+
+:::demo Text mixed component
+
+text/mixed
+
+:::
+
+## Text API
```

Comment:
```suggestion
## API
```

Reply:
ElButton supports all experimental features, so I think I'm going to learn button, are you sure I should change?

[reference el-button doc](https://github.com/element-plus/element-plus/blob/dev/docs/en-US/component/button.md)

Reply:
The ElButton page contains two components, but there is only one

Reply:
Got it.